pages/css/index.css

```diff
@@ -0,0 +1,70 @@
+/* Add some fun and cool CSS styles here */
+body {
+    font-family: 'Roboto', sans-serif;
+    background-color: #f5f5f5;
+    margin: 0;
+    padding: 0;
+  }
+  
+  header {
+    background-color: #2893a1;
+    color: #fff;
+    padding: 10px;
+    text-align: center;
+  }
+  
+  nav ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+  
+  nav li {
+    display: inline-block;
```

Comment:
To make reading easy and avoid possible confusion, I suggest keeping all CSS rules at the same indentation level.